cwltool/load_tool.py

```diff
@@ -165,8 +165,8 @@ def validate_document(document_loader,  # type: Loader
             r"^(?:cwl:|https://w3id.org/cwl/cwl#)", "",
             workflowobj["cwlVersion"])
     else:
-        _logger.warning("No cwlVersion found, treating this file as draft-2.")
-        workflowobj["cwlVersion"] = "draft-2"
+        _logger.warning("No cwlVersion found, treating this file as v1.0")
```

Comment:
@tetron No such fallback should be provided? 
https://github.com/common-workflow-language/cwltool/issues/482#issuecomment-316705149